roles/fio_distributed/templates/client.yaml

```
@@ -33,6 +33,22 @@ spec:
             value: "{{ elasticsearch.index_name | default("ripsaw-fio") }}"
           - name: es_verify_cert
             value: "{{ elasticsearch.cert_verify | default("true") }}"
+          - name: parallel
+            value: "{{ elasticsearch.parallel | default("false") }}"
+{% endif %}
+{% if prometheus is defined %}
```

I think the second conditional is not required here. All values can be set under the first conditional.
  
```suggestion
{% if prometheus is defined %}
          - name: prom_es
            value: "{{ prometheus.server }}"
          - name: prom_port
            value: "{{ prometheus.port }}"
          - name: prom_parallel
            value: "{{ prometheus.parallel | default("false") }}"
          - name: prom_token
            value: "{{ prometheus.prom_token }}"
          - name: prom_url
            value: "{{ promethues.prom_url }}"
{% endif %} 
```

In addition, prom_token and prom_url should be replaced by prometheus.prom_token and prometheus.prom_url

---

good catch. thanks!